tests/conftest.py

```diff
@@ -7,6 +7,7 @@ def pytest_configure(config):
     from django.conf import settings
 
     os.environ.setdefault("NINJA_SKIP_REGISTRY", "True")
+    os.environ.setdefault("DJANGO_ALLOW_ASYNC_UNSAFE", "True")
```

Comment:
@QtiQla Please what does this do?

Reply:
Oh, my bad. I was not able to create test data in model to test async methods without this variable

Reply:
Is it still useful? if not you can remove it.

Reply:
Yes, it is. Without this var async tests  doesn't work, they get exeption like "You cannot call this from an async context - use a thread or sync_to_async", i couldn't fix it without using the variable above